distributed/deploy/local.py

```diff
@@ -27,11 +30,10 @@ class LocalCluster(object):
     ----------
     n_workers: int
         Number of workers to start
+    processes: bool
+        Whether to use processes (True) or threads (False).  Defaults to True
```

Comment:
Yes.  Anecdotally I've observed that this is more often what people seem to want and expect.

Changing this would be very convenient for us though.